packages/react/react/src/hooks/useCombinedRefs.ts

```diff
@@ -18,17 +16,16 @@ type CallbackRef<T> = (ref: T | null) => void;
  *   return <div ref={ref} />;
  * })
  */
-export function useCombinedRefs<T>(...refs: Array<Ref<T> | CallbackRef<T>>): Ref<T> {
-  return useCallback(
-    (value: T) => {
-      for (const ref of refs) {
+export const useCombinedRefs = <T>(...refs: Array<Ref<T>>): RefCallback<T> =>
```

Comment:
Why did you change function declaration into arrow function?

Reply:
I think specifying the return type to be `RefCallback<T>`, rather than `Ref<T>`, exposes the actual implementation, making us hard to change the internal implementation. Were there some backgrounds to make return types more specific?